miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcherSpec.kt

```diff
@@ -0,0 +1,333 @@
+package com.rakuten.tech.mobile.miniapp.js
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.gson.Gson
+import com.rakuten.tech.mobile.miniapp.TEST_CALLBACK_ID
+import com.rakuten.tech.mobile.miniapp.TEST_MA_ID
+import com.rakuten.tech.mobile.miniapp.TestActivity
+import com.rakuten.tech.mobile.miniapp.display.WebViewListener
+import com.rakuten.tech.mobile.miniapp.errors.MiniAppSecureStorageError
+import com.rakuten.tech.mobile.miniapp.storage.MiniAppSecureStorage
+import com.rakuten.tech.mobile.miniapp.storage.StorageState
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.amshove.kluent.When
+import org.amshove.kluent.calling
+import org.amshove.kluent.itReturns
+import org.amshove.kluent.shouldBe
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+
+@ExperimentalCoroutinesApi
+@RunWith(AndroidJUnit4::class)
+@Suppress("LargeClass")
+class MiniAppSecureStorageDispatcherSpec {
+    private val TEST_KEY = "key"
+    private val TEST_VALUE = "value"
```

Comment:
We can resolve this warning from AS:
![Screen Shot 2022-05-19 at 14 46 21](https://user-images.githubusercontent.com/66930373/169219087-7bd6da23-deb8-48bd-a4bf-29948dedec21.png)


So - we can move those fields into a companion object e.g.

```
private companion object {
        private const val TEST_KEY = "key"
        ...
}
```